twake/backend/core/src/Twake/Discussion/Entity/Message.php

```diff
@@ -478,8 +478,106 @@ public function setIncrementAtTime($increment_at_time)
         $this->increment_at_time = $increment_at_time;
     }
 
+    public function getStringMessageType() {
+        //0 from user, 1 from application, 2 from system
+        switch($this->getMessageType()) {
+            case 0: return "message";
+            case 1: return "thread";
+            case 2: return "event";
+            default:
+                return "message";
+        }
+    }
```

Comment:
This is not exactly right it should be:
*$this->getMessageType() === 0*: type="message", subtype=null
*$this->getMessageType() === 1*: type="message", subtype="application"
*$this->getMessageType() === 2*: type="message", subtype="system"

Reply:
Updated :)